book/src/api-vc-sig-header.md

```diff
@@ -0,0 +1 @@
+# Signature Header
```

Comment:
Does here suppose to have some content?

Reply:
Whoops, that file mysteriously ended up in my branch, have removed it now